action.yml

```diff
@@ -0,0 +1,15 @@
+name: Clean dependencies
+author: Poulinho
+description: 'An easy GA to clean your dependencies before generate artifact'
+branding:
+  icon: 'check'
+  color: 'blue'
+inputs:
+  path:
```

Comment:
```suggestion
  files:
```

Reply:
IMO path make more sens because we are doing `rm -rf /the/path` and not giving the possibility to give multiple files path

Reply:
I'm thinking the same thing as @zalexki putting in input `files` could confuse the user.

Reply:
My point is to allow several files to be deleted actually, as `rm` accepts several arguments. I chose `files` as folders are a special type of file in a filesystem, but `paths` would be valid too.

Reply:
All right, so can we just say this discussion can be resolved?

Reply:
I would write this argument in plural form at least.

Reply:
I just made the change.